WooCommerce/Classes/Extensions/WKWebView+Authenticated.swift

```diff
@@ -1,12 +1,29 @@
 import Alamofire
 import Foundation
 import WebKit
+import WordPressAuthenticator
```

Comment:
super nit: would be nice to just import the class/protocol that we need, probably just `WordPressOrgCredentials`

Reply:
Updated in 4128d7473d0dc9b0b43b7991133c17cf2d0d5c59.